### ISSUE TITLE
508 Accessibility: Color contrast - panel on background

### ISSUE THREAD

**Issue by grossb1** [department-of-veterans-affairs/va.gov-team#21854]
When reviewing the Project Cheetah workflow for 508 accessibility issues the following item was flagged for failing a color contrast test.
The panel toward the bottom of the linked page didn't not have a good enough contrast value when compared against the white background.  The contrast was fine under normal conditions but when tested with a color-blindness tool the contrast tests were failing.

.../health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/facility

Steps to reproduce:
Navigate to the above page
Activate your colorblind testing tool (I used the Colorblinding chrome plugin)
Sample the hex color of the panel toward the bottom of the page (You can use something like Pixie to do this)
Sample the hex color of the background, it should be #FFFFFF
Go to a website like https://whocanuse.com/ and enter the hex values sampled above
Notice that the contrast test fails.

@outerpress

**Comment by 1Copenut** [comment]
Please see https://github.com/department-of-veterans-affairs/va.gov-team/issues/21848#issuecomment-805969640 for this issue.
